libUnderworld/StgFEM/Utils/src/SemiLagrangianIntegrator.c

```diff
@@ -467,7 +482,10 @@ void BicubicInterpolator( FeVariable* feVariable, double* position, double* delt
       for( y_i = 0; y_i < 4; y_i++ )
          for( x_i = 0; x_i < 4; x_i++ ) {
             gNode_I = x_0 + x_i + ( y_0 + y_i ) * nNodes[0];
-            if( !Mesh_GlobalToDomain( feMesh, MT_VERTEX, gNode_I, &lNode_I ) ) abort();
+            if( !Mesh_GlobalToDomain( feMesh, MT_VERTEX, gNode_I, &lNode_I ) ){
+              printf("Error in BicubicInterpolator(), trying to build an interpolation to position (%g, %g) using node %d, a non domain node, in interpolator.\n", position[0], position[1], gNode_I);
```

Comment:
This is better but it really needs to be a `Journal_Firewall()`.  `abort()` will just kill the kernel, and the error message won't (i think) appear in the notebook.